weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PromptPage/types.ts

```diff
@@ -0,0 +1,31 @@
+// TODO: lock this down
```

Comment:
Oh i missed this as it relates to my comment on `CallChat`. This is more what i was looking for. It seems like everything in this directory (except `PromptTab`) could be in it's own dir: `LLMMessageList`